dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -1,137 +1,518 @@
 <template>
-  <Card class="col-12">
+  <Card class="col-12 page-wrapper-card">
     <template #title>Create EU Taxonomy Dataset for a Financial Company/Service</template>
     <template #content>
-      <FormKit
-        v-model="formInputsModel"
-        :actions="false"
-        type="form"
-        id="createEuTaxonomyForFinancialsForm"
-        @submit="postEuTaxonomyDataForFinancials"
-        #default="{ state: { valid } }"
-      >
-        <h4>General Information</h4>
-        <FormKit
-          type="text"
-          name="companyId"
-          label="Company ID"
-          placeholder="Company ID"
-          :model-value="companyID"
-          disabled="true"
-        />
-        <FormKit type="text" name="reportingPeriod" label="Reporting Period" placeholder="Reporting Period" />
-        <FormKit type="group" name="data" label="data">
+      <div class="grid uploadFormWrapper">
```

Comment:
After loading the page an error regarding the updateObjectUtils is observed in the console
<img width="361" alt="ErrorUpdateObjectUtils" src="https://user-images.githubusercontent.com/104447806/227974466-278d1783-dfa5-4506-a6e6-cf6700389264.PNG">

Reply:
done

Reply:
ok